flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyMessage.java

```diff
@@ -841,6 +844,52 @@ public String toString() {
         }
     }
 
+    /** Message to notify producer about new buffer size. */
+    static class NewBufferSize extends NettyMessage {
+
+        private static final byte ID = 10;
+
+        final long bufferSize;
```

Comment:
I don't sure that we should use `long` for the buffer size. As I can see now `NetworkBuffer#currentSize` or `MemorySegment#size` is `int` so perhaps I should migrate from `long` to `int`

Reply:
Yes, you are right. `long` doesn't make sense here.